var/spack/repos/builtin/packages/util-linux-uuid/package.py

```diff
@@ -46,4 +46,6 @@ def configure_args(self):
             '--enable-libuuid',
             '--disable-bash-completion',
         ]
+        if self.spec.satisfies('%intel'):
+            config_args.append('CFLAGS=-restrict')
```

Comment:
Is there a bug report or something we can put in a comment in case someone stumbles over this in the future?

Reply:
Yes, will try to get to that tomorrow.

Reply:
@michaelkuhn I've added a comment with the issue number.  Let me know if there are any other changes you'd like to see.